docker-entrypoint.sh

```diff
@@ -8,7 +8,7 @@ SQLD_DB_PATH="${SQLD_DB_PATH:-iku.db}"
 SQLD_PG_LISTEN_ADDR="${SQLD_PG_LISTEN_ADDR:-"0.0.0.0:5432"}"
 SQLD_HTTP_LISTEN_ADDR="${SQLD_HTTP_LISTEN_ADDR:-"0.0.0.0:8080"}"
 
-SQLD_HTTP_AUTH="${SQLD_HTTP_AUTH:-"always"}"
+# SQLD_HTTP_AUTH="${SQLD_HTTP_AUTH:-"always"}"
```

Comment:
leftovers from debugging?

Reply:
yeah, but actually what is that? I can't launch the server with this set to its default. what's that `always`?